packages/dashboard/src/serviceWorker.ts

```diff
@@ -15,9 +15,7 @@ const isLocalhost = Boolean(
     // [::1] is the IPv6 localhost address.
     window.location.hostname === '[::1]' ||
     // 127.0.0.1/8 is considered localhost for IPv4.
-    window.location.hostname.match(
```

Comment:
Was anything else changed aside from formatting?

Reply:
Nope, I copied and paste the original version in the main branch